adding-module.rst

```diff
@@ -62,15 +63,16 @@ structure. Open the ``mymodule.c`` file and put this code inside:
     #include "portmodules.h"
 
 
-    STATIC const mp_map_elem_t mymodule_globals_table[] = {
+    STATIC const mp_map_elem_t mymodule_globals_table[] = 
+    {
```

Comment:
We don't put braces on separate lines. Also, it would be nice to submit any code style changes in a separate pull request.